### DISCUSSION TITLE
How can i add serving signture to tflite model.

### DISCUSSION THREAD

**Discussion by linuxlonelyeagle** [iree-org/iree#13113] [General]
You can see the following code,but this model is used for the tf model.If I am using the tflite model, how should I solve this problem?
```
import tensorflow.compat.v2 as tf
loaded_model = tf.saved_model.load('/path/to/downloaded/model/')
call = loaded_model.__call__.get_concrete_function(
         tf.TensorSpec([1, 224, 224, 3], tf.float32))
signatures = {'predict': call}
tf.saved_model.save(loaded_model,
  '/path/to/resaved/model/', signatures=signatures)
```

**Comment by GMNGeoffrey** [comment]
@rsuderman might be able to help here

**Comment by ScottTodd** [comment]
Does this help? https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/tensorflow_hub_import.ipynb#scrollTo=CedNRSQTOE7C

**Comment by linuxlonelyeagle** [comment]
Before you answer this question, I have actually found an alternative to my problem, at the very beginning I was trying to get the tflite model f32 data to convert to i8 data (my coprocessor only supported i8 before), I referred to the ft model approach, I now think I have a problem with my thinking, so I raised this issue. thank you very much for your answers.
